requirements.txt

```
@@ -1,3 +1,3 @@
-eth-ape>=0.5.1
+eth-ape>=0.5.2
```

Commit after release of eth-ape

```suggestion
eth-ape>=0.5.4
```

---

Try adding this to re-trigger the build and see if that fixes the Pydantic validation